src/highlight/highlightUtil.ts

```diff
@@ -3,8 +3,8 @@ import { Annotation, AnnotationHighlight, Position, Shape, Type } from '../@type
 export const getBoundingRect = (shapes: Shape[]): Shape => {
     let minX = Number.MAX_VALUE;
     let minY = Number.MAX_VALUE;
-    let maxX = Number.MIN_VALUE;
-    let maxY = Number.MIN_VALUE;
+    let maxX = Number.NEGATIVE_INFINITY;
```

Comment:
Can this lead to a situation where negative values are saved to the DB? I don't think that was ever planned for, so it may not be handled properly.

Reply:
No this shouldn't because prior to even getting set to the store in creator `staged` it transforms the shape relative to the container and at that point they get changed to percentages

Reply:
Can we validate that by creating some tests that pass shapes with negative values to that transform logic?